internal/patch/grpc-go-insecure/go.mod

```diff
@@ -0,0 +1,6 @@
+module google.golang.org/grpc/credentials/insecure
+
+go 1.16
+
+// Should match the version we replace on the main go.mod
```

Comment:
should this comment also be mentioned in the top-level go.mod of the repo (next to the line that pins `google.golang.org/grpc`?)

Reply:
Makes sense, I added a comment